plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/StartMojo.java

```diff
@@ -265,9 +265,15 @@ protected SwarmExecutor executor(final Path appPath, final String name,
                 .sorted()
                 .collect(Collectors.toList())));
 
-        fractions.addAll(this.additionalFractions.stream()
-                                 .map(f -> FractionDescriptor.fromGav(FractionList.get(), f))
-                                 .collect(Collectors.toSet()));
+        this.fractions.forEach(f -> {
```

Comment:
I think it's worth renaming the local `fractions` variable to prevent confusion between it and `this.fractions`

Reply:
+1